BUILDING.md

```diff
@@ -63,27 +62,23 @@ correct version of `runc` installed.
 
 For the quick and dirty installation, you can use the following:
 
-    go get github.com/opencontainers/runc
-
-This is not recommended, as the generated binary will not have version
```

Comment:
why remove this part?

Reply:
Using `go get` to install things is confused. It will accidentally touch your `go.mod`. After go1.16, it will be improved, but the command is not right then, it needs to be `go get github.com/opencontainers/runc@version`.

So currently keeping the old things from GOPATH doesn't feel beneficial.